src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
I see [overwriteMethod](http://chaijs.com/api/plugins/#method_overwritemethod) is available for plugins, but I'm not sure how we could tell whether the object under test (normally a string in our case) is supposed to be a selector or another kind of object to which the normal `exist` should apply.  The chain you suggested (or any version of allowing either or) would not raise an exception in the case where the selector doesn't exist (because the string itself does exist).

We'd have to somehow be able to tell the difference between an object being tested with `chai-webdriverio` (i.e. a selector), and any other object.  Which is probably why they have the `dom` flag in `chai-webdriver`, I guess??  But I don't feel like implementing that right now, haha.

Another option would be to change the syntax to use `$` at which point we'd have a recognizable object... i.e.:

```javascript
  expect($('.some-selector')).to.exist;
  // Now we can check that the object is a... whatever $ returns :)
```